static/js/containers/ChannelPage.js

```diff
@@ -123,13 +127,26 @@ export class ChannelPage extends React.Component<ChannelPageProps> {
             <title>{formatTitle(channel.title)}</title>
             <CanonicalLink match={match} />
           </MetaTags>
-          <div className="post-list-title">
-            <PostSortPicker
-              updatePickerParam={updatePostSortParam(this.props)}
-              value={qs.parse(search).sort || POSTS_SORT_HOT}
-            />
-          </div>
-          {renderPosts()}
+          <Grid className="main-content two-column channel-page">
+            {!isAbout || !isMobileWidth() ? (
```

Comment:
We might want to consider something like [react-media](https://www.npmjs.com/package/react-media) to handle hiding components responsively. Since this has the same limitation as the drawer currently does, you can take-it or leave-it.

Reply:
It might be useful but I'd like to deal with it in a separate PR so the drawer can use it too